OpenFeature/src/main/java/dev/openfeature/sdk/Value.kt

```diff
@@ -73,14 +74,21 @@ object ValueSerializer : JsonContentPolymorphicSerializer<Value>(Value::class) {
     }
 }
 
+@SuppressLint("SimpleDateFormat")
```

Comment:
out of curiosity, why do we need this?

Reply:
It complains in Android Studio that I'm not using a Locale to construct the object.